flink/v1.20/flink/src/main/java/org/apache/iceberg/flink/source/IcebergSource.java

```diff
@@ -482,6 +482,10 @@ public IcebergSource<T> build() {
       }
 
       contextBuilder.resolveConfig(table, readOptions, flinkConfig);
+      contextBuilder.exposeLocality(
+          SourceUtil.isLocalityEnabled(table, flinkConfig, exposeLocality));
+      contextBuilder.planParallelism(
```

Comment:
Quick question: I see `contextBuilder.resolveConfig(table, readOptions, flinkConfig)` above already sets the planParallelism via `.planParallelism(flinkReadConf.workerPoolSize())`, is that not sufficient?